cli/flags.rs

```diff
@@ -154,6 +154,16 @@ To get help on the another subcommands (run in this case):
 Includes versions of Deno, V8 JavaScript Engine, and the TypeScript
 compiler.",
         ),
+    ).subcommand(
+      SubCommand::with_name("bundle")
+        .setting(AppSettings::DisableVersion)
+        .about("Bundle module and dependnecies into single file")
+        .long_about(
+          "Fetch, compile, and output to a single file a module and its dependencies.
+"
```

Comment:
Can you give an example command here that can be copied and pasted, so people can try it out quickly?

Maybe use a remote URL, like 
```
deno bundle https://deno.land/std/examples/colors.ts colors_bundle.js
```